osf/models/node.py

```diff
@@ -1695,7 +1695,7 @@ def register_node(self, schema, auth, data, parent=None):
             node_contained = node_relation.child
             # Register child nodes
             if not node_relation.is_node_link:
-                registered_child = node_contained.register_node(  # noqa
+                node_contained.register_node(  # noqa
```

Comment:
Why did this get changed in this PR? Also, is the noqa comment in there because flaking was failing due to the unused variable? If so, can it be removed (if we keep this change)?

Reply:
Yes, this is an unused variable which will cause flake 8 error. Not sure why it doesn't raise that error before. Since that variable isn't used anywhere, better to remove it.

Reply:
I think you should be able to remove the `#noqa` then

Reply:
fixed in 991d290